rpc/src/lib.rs

```diff
@@ -38,3 +38,24 @@ pub mod mayastor {
 
     include!(concat!(env!("OUT_DIR"), "/mayastor.rs"));
 }
+
+#[allow(dead_code)]
+#[allow(clippy::type_complexity)]
+#[allow(clippy::unit_arg)]
+#[allow(clippy::redundant_closure)]
+#[allow(clippy::upper_case_acronyms)]
+pub mod mayastorv1 {
+
+    #[derive(Debug)]
+    pub enum Error {
```

Comment:
what's this error with only 1 variant needed for? Why in this module? This also seems very generic.

Reply:
I guess this is not needed. Removing it.